glitch/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+services:
+
+  #PHP Service
+  glitch:
+    image: fishnet/glitch
+    container_name: glitch-php
+    restart: unless-stopped
+    tty: true
+    working_dir: /var/www
+    volumes:
+      - ./:/var/www
+      - node_modules:/var/www/node_modules
+      - ./config/php.ini:/usr/local/etc/php/conf.d/settings.ini
+    networks:
+      - glitch-network
+
+  webserver:
+    image: nginx:alpine
+    container_name: glitch-nginx
+    restart: unless-stopped
+    ports: 
+      - "80:80"
+    volumes:
+      - ./:/var/www
+      - ./config/nginx.conf:/etc/nginx/conf.d/default.conf
+      - node_modules:/var/www/node_modules
+    networks:
+      - glitch-network
+    depends_on:
+      - glitch 
+
+#Docker Networks
+networks:
+  glitch-network:
+    driver: bridge
+
+#Volumes
+volumes:
+    dbdata:
+      driver: local
```

Comment:
Shouldn't this database volume also be removed?

Reply:
They definitely don't need to be there, my bad. I'll edit in a sec